requirements.txt

```diff
@@ -1,26 +1,26 @@
-baytune>=0.2.3,<0.3.0
-bokeh>=1.0.2
-composeml>=0.3.0
-dask[bag]>=1.0.0
-evalml==0.16.1
-featuretools>=0.16.0,<1.0.0
-featuretools_tsfresh_primitives>=0.1.3
-findspark==1.4.2
-graphviz>=0.10.1
-ipython>=5.8.0
-jupyter>=1.0.0
-lightgbm>=2.2.2
-matplotlib>=2.2.3
-notebook>=6.0.0
-pandas==1.1.5
-psutil>=5.4.8
-pyarrow>=0.11.1
-pyspark==3.0.0
-requests>=2.21.0
-scikit-learn>=0.20.2
-seaborn>=0.9.0
-TPOT==0.10.2
-tqdm>=4.19.2
-tsfresh>=0.11.1
-umap-learn>=0.3.7
-xgboost>=0.81
+# baytune>=0.2.3,<0.3.0
+# bokeh==2.4.2
+# composeml>=0.3.0
+# dask[bag]>=1.0.0
+evalml==0.39.0
```

Comment:
I agree that we should try to minimize this as much as possible. Maybe we just need evalml and featuretools?

Reply:
My plan here is to pin the necessary ones, and drop everything else out. As I work through the notebooks I will add in the dependencies that become necessary until we get to the final list of what is required...and hope for no conflicts along the way.

There will be a few others that are necessary beyond the three you mention - I know `bokeh` is used by some notebooks to plot results.